tests/jenkins/TestDataPrepperDistributionArtifacts.groovy

```diff
@@ -3,35 +3,68 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import jenkins.tests.BaseJobPipelineTest
+
+import jenkins.tests.BuildPipelineTest
 import org.junit.Before
 import org.junit.Test
 
-class TestDataPrepperDistributionArtifacts extends BaseJobPipelineTest {
+class TestDataPrepperDistributionArtifacts extends BuildPipelineTest {
 
     @Before
     void setUp() {
         super.setUp()
 
-        binding.setVariable('VERSION', '0.22.1')
-        binding.setVariable('BRANCH', 'refs/tags/0.22.1')
+        // job parameters
+        def version = '0.22.1'
+        def branch = 'refs/tags/0.22.1'
 
+        // Variables for Data-prepper-Distribution-Artifacts-job
+        def workspace = '/tmp/workspace'
+        binding.setVariable('VERSION', version)
+        binding.setVariable('BRANCH', branch)
         binding.setVariable('JOB_BASE_NAME', 'data-prepper-distribution-artifacts')
-        binding.setVariable('BUILD_NUMBER', '7654321')
+        binding.setVariable('BUILD_NUMBER', '51')
         binding.setVariable('WORKSPACE', '/tmp/workspace')
-
         binding.setVariable('ARTIFACT_BUCKET_NAME', 'job-s3-bucket-name')
 
         helper.registerAllowedMethod('checkout', [Map], {})
-        helper.registerAllowedMethod('signArtifacts', [Map], {})
-        helper.registerAllowedMethod('uploadToS3', [Map], {})
-        helper.registerAllowedMethod('printArtifactDownloadUrlsForStaging', [Map], {})
-        helper.registerAllowedMethod('postCleanup', [], {})
+
+        def artifactsPath = "$workspace/release/archives/linux/build/distributions"
+        helper.addShMock("find ${artifactsPath} | sed -n \"s|^${artifactsPath}/||p\"") { script ->
+            return [stdout: "tar_dummy_1_artifact_1.0.0.tar.gz tar_dummy_2_artifact_1.0.0.tar.gz", exitValue: 0]
+        }
+
+
+        // Variables for signArtifacts library
+        binding.setVariable('GITHUB_BOT_TOKEN_NAME', 'github_bot_token_name')
+
+        binding.setVariable('SIGNER_CLIENT_ROLE', 'dummy_signer_client_role')
+        binding.setVariable('SIGNER_CLIENT_EXTERNAL_ID', 'signer_client_external_id')
+        binding.setVariable('SIGNER_CLIENT_UNSIGNED_BUCKET', 'signer_client_unsigned_bucket')
+        binding.setVariable('SIGNER_CLIENT_SIGNED_BUCKET', 'signer_client_signed_bucket')
+
+        helper.registerAllowedMethod("git", [Map])
+        helper.registerAllowedMethod("withCredentials", [Map])
+
+
+        // Upload to s3
```

Comment:
Same suggestion as above, the library is `uploadToS3`, so I like the idea of having `setupUploadToS3()` that can be used in tests.